antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -46,22 +46,22 @@
         cancel.style.display = 'block'
         index
           .search(search.value, {
-            hitsPerPage: 10,
+            hitsPerPage: 5,
           })
           .then((results) => {
             const hits = results.hits
             const data = hits.reduce((data, hit) => {
               const d = {}
               d.url = hit.url
               var section = hit.hierarchy.lvl0
-              if (hit.hierarchy.lvl6 !== null) section = section + ' [' + hit.hierarchy.lvl6 + ']'
+              if (hit.hierarchy.lvl6 !== null) section = section + '/' + hit.hierarchy.lvl6
               var breadcrumbs = Object.values(hit.hierarchy)
                 .slice(1)
                 .filter((lvl) => lvl !== null)
-                .join(' &raquo; ')
+                .join(' / ')
 
               d.breadcrumbs = ((breadcrumbs !== '') ? breadcrumbs : section)
-              d.snippet = hit._snippetResult.content.value + '...'
+              d.snippet = hit._snippetResult.content.value.split('&quot;').join('') + '...'
```

Comment:
Seems that `hit._snippetResult` can be undefined, are we using `d.snippet`?

Reply:
Yes, that makes sense, I altered the code to take care of it so it doesn't split for undefined values. Yes, we use `d.snippet` within the `summary` tag.